src/ui/public/utils/migrateLegacyQuery.ts

```diff
@@ -25,11 +25,10 @@ import { has } from 'lodash';
  * @param query - a legacy query, what used to be stored in SearchSource's query property
  * @return Object
  */
-export function migrateLegacyQuery(query) {
-
+export function migrateLegacyQuery(query: object | string): object {
```

Comment:
Do we use `string` sometimes here?  Looks like lodash's has only takes `object`.  Looks like that's the reason for the build failure:

```
 debg src/ui/public/utils/migrateLegacyQuery.ts:34:3 - error TS2322: Type 'string | object' is not assignable to type 'object'.
```